README.md

```diff
@@ -385,7 +385,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Arista](https://www.arista.com/en/careers/university/engineering-internship) | Multiple Locations in US | Software Engineer Intern |
 | [Expedia Group](https://careers.expediagroup.com/jobs/?filter%5Bcategory%5D=&filter%5Bcountry%5D=&filter%5Bstate%5D=&filter%5Bcity%5D=)| Multiple Locations in US | [Software Development Engineer Intern (all stacks)](https://careers.expediagroup.com/jobs/?keyword=2023+Intern+-+Software+Development+Engineer+%28all+stacks%29&&filter[country]=United+States) |
 | [Clever](https://clever.com/about/careers/detail?gh_jid=4525415&gh_src=cfcca5b51us) | Multiple Locations in US | Software Engineering - Intern |
-
+| [Tesla](https://www.tesla.com/careers/search/?query=Internship&department=2&site=US) | Palo Alto, CA | [Software Engineering (Internship)](https://www.tesla.com/careers/search/job/software-engineering-internship-spring-2023-136642), [Full Stack Engineer (Internship)](https://www.tesla.com/careers/search/job/full-stack-engineer-autopilot-ai-tooling-internship-152104)|
```

Comment:
Looks like both are spring internships. I though this repo should only contain `Summer 2023` internships